.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,41 @@
+name: PR Test Label Manager
```

Comment:
[note] RDK link: https://github.com/viamrobotics/rdk/blob/main/.github/workflows/pr-labeler.yml

Reply:
before merging `qaworkflow` needs to be changed for this file as well right?

Reply:
Yep, all references to qaworkflow will be removed before merge

Reply:
Mind please posting the diff?